src/main/groovy/pluggable/scm/SCMProviderHandler.groovy

```diff
@@ -21,8 +20,11 @@ public class SCMProviderHandler {
   *   SCM provider id. The method uses the SCM Provider data store to infer
   *   the SCM providers configuration.
   * @param scmProviderId - the unique id of the SCM provider.
+  * @param variableBindings
```

Comment:
What is listener used for? Can we remove it? Please can we also add a description to the variablesBindings param?